hack/kpng-local-up.sh

```diff
@@ -4,10 +4,9 @@
 # TODO Replace with 1.22 once we address 
 #: ${KIND:="kindest/node:v1.21.1@sha256:69860bda5563ac81e3c0057d654b5253219618a22ec3a346306239bba8cfa1a6"}
 : ${KIND:="kindest/node:v1.22.0@sha256:b8bda84bb3a190e6e028b1760d277454a72267a5454b57db34437c34a588d047"}
-: ${IMAGE:="jayunit100/kpng:ipvs"}
+: ${IMAGE:="jayunit100/kpng:2"}
 : ${PULL:=IfNotPresent}
-: ${BACKEND:=nft}
-
+: ${BACKEND:=iptables}
```

Comment:
it think this change is not required ;)

Reply:
nft doesnt work, iptables more common so seemed like the right thing to do?